DeviceComunication/DeviceComunication_CZ.md

```diff
@@ -1048,3 +1092,9 @@ Zařízení vypočítá počet měření, které je potřeba udělat před odesl
 
 Zařízení pak bude čekat mezi měřeními 4 minuty a po každých dvou měření odešle zprávu. Measure
 zprávy se tedy budou odesílat každých 8 minut.
+
+## Zjednodušená implementace potvrzování
+Protože zařízení ve výchozím stavu vyžaduje potvrzení některých zpráv 
+(určuje [5.byte v hlavičce](#5byte---nb-iot)), je nutné implementovat alespoň 
+základní komunikaci ze serveru do zařízení. Na tyto zprávy je možné zjednodušeně 
+odpovídat staticky payloadem `0000000001FF00` který zajistí potvrzení přijaté zprávy.
```

Comment:
Pak bych jeste pridal k restart zprave informaci o tom jak ji deduplikovat (musi se to delat pomoci toho pocitadla).

Ted jsem si uvedomil ze to nejde.  Rekneme ze me na server prijde restart a ma pocitadlo zprav  0 a pocet odeslani 0. A pak prijde druhy ktery ma take pocitadlo 0 a pocet odeslani 1 tak ty server nemůže rozlisit jestli je to stejny restart a nebo jiny restart ktery se nepodarilo napoprvé odeslat.

Koukam ted ale na ten payload a ve 12.byte restart zpravy je pocitadlo restartu od posledního hard restartu tak tim by se to dalo deduplikovat. 

Tak to bych dopsal do te dokumentace ze 12.byte slouží k deduplikaci.

Do tech vylepseni firmwaru co si nekde pises bych napsal ze by bylo dobre pocitadlo zprav pocitat od hard restartu a ne od restartu.

Reply:
Doplněn text:

Při restartu se nuluje 0.byte v hlavičce udávající počet odeslaných zpráv a nelze jej tedy u tohoto typu zprávy využít pro rozpoznání dulicitní zprávy. Pro deduplikaci lze částečně využít 12.byte s údajem o počtu restartů s vyjímkou případu hard restartu (0x00 ve 13.byte) při kterém se toto počitadlo nuluje.